test/terra/extensions/test_wrappers.py

```diff
@@ -64,25 +60,11 @@ def _map_and_test(self, cfunc, qobj):
 
     def test_mappable_qasm(self):
         """Test that the qasm controller can be mapped."""
-        cfunc = qasm_controller_execute()
+        cfunc = aer_controller_execute()
         sim = QasmSimulator()
         fqobj = self._create_qobj(sim)
         self._map_and_test(cfunc, fqobj)
 
-    def test_mappable_statevector(self):
-        """Test that the statevector controller can be mapped."""
-        cfunc = statevector_controller_execute()
-        sim = StatevectorSimulator()
-        fqobj = self._create_qobj(sim)
-        self._map_and_test(cfunc, fqobj)
-
-    def test_mappable_unitary(self):
-        """Test that the unitary controller can be mapped."""
-        cfunc = unitary_controller_execute()
-        sim = UnitarySimulator()
-        fqobj = self._create_qobj(sim)
-        self._map_and_test(cfunc, fqobj)
-
```

Comment:
I didn't notice any relevant methods on `StatevectorSimulator` or `UnitarySimulator` being removed; should these test cases remain for now?

Reply:
Oh, on second thoughts - were they removed from the C++?  I probably didn't notice if they were, since there were several whole files removed.

Reply:
Yeah these were testing the pickling of the additional pybind wrapper that were removed.